### PR TITLE
fix(1718): allow user to set test and coverage url

### PR DESCRIPTION
**Pull request by klu909**
## Context

Currently there's no way to set test and coverage urls even though we can set summary.

## Objective

Allow user to set test and coverage urls .

## References

https://github.com/screwdriver-cd/screwdriver/issues/1718

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/build-banner/component.js [hunk 1]
```diff
@@ -77,20 +77,29 @@ export default Component.extend({
 
     // override coverage info if set in build meta
     if (buildMeta && buildMeta.tests) {
-      const coverage = String(buildMeta.tests.coverage);
-      const tests = String(buildMeta.tests.results);
-      let { coverageInfo } = this;
+      const { coverage, coverageUrl, results: tests, resultsUrl: testsUrl } = buildMeta.tests;
+      const regex = /^.+\/pipelines\/\d+\/builds\/\d+/;
```
Comment (tkyi) [inline app/components/build-banner/component.js]: maybe a more specific variable name would be better?
`BUILD_URL_REGEX`?

Comment (ldong) [reply app/components/build-banner/component.js]: I agree the variable names could be more self-explanatory. @klu909 Please see @tkyi 's comment.

**Comment by klu909** [comment]
> Approved. Please create a sample pipeline to demo the usage of it.
> I'd like to checkout how to use it prior roll out, but we can test on staging as well.
test pipeline https://cd.screwdriver.cd/pipelines/3910/events

**Comment by adong** [comment]
Sample yaml 
```
  
shared:
    image: node:12

jobs:
    main:
        requires: [~pr, ~commit]
        steps:
            - generate-artifacts:  find ./artifacts -print | sed -e 's;[^/]*/;|____;g;s;____|; |;g'
            - copy-to-artifacts-dir: echo $SD_ARTIFACTS_DIR && cp -r ./artifacts $SD_ARTIFACTS_DIR
            - set-coverage-and-test-results: |
                meta set tests.coverage 100
                meta set tests.results 10/10
                meta set tests.coverageUrl /artifacts/jump.png   # key  
                meta set tests.resultsUrl /artifacts/steps.json  # key 
```
From 
https://github.com/klu909/artifacts-test/blob/7e4325b3d289741db0869557b3d186b944c71467/screwdriver.yaml#L13-L14

**Pull request merged**